packages/components/src/mobile/alt-text-settings/styles.scss

```diff
@@ -0,0 +1,32 @@
+.altTextEditor {
+	padding: 14px 0;
+	height: 80px;
+}
+
+.altTextEditorDark {
+	color: $white;
+}
+
+.altTextFootnote {
+	font-size: 12px;
+	color: $gray-text-min;
+	padding: 8px 16px 24px;
+}
+
+.altTextLink {
+	color: $blue-50;
+}
+
+.altTextLinkDark {
+	color: $blue-30;
+}
+
+.horizontalBorder {
+	border-top-width: $border-width;
+	border-bottom-width: $border-width;
+	border-color: $light-gray-400;
+}
+
+.horizontalBorderDark {
+	border-color: $gray-70;
```

Comment:
Do we not need not need 
```
border-top-width: $border-width;
border-bottom-width: $border-width;
```
here?

Reply:
👋 @enejb, thank you for your comments and looking over the code here. :D 

I'm using [the usePreferredColorSchemeStyle hook](https://github.com/WordPress/gutenberg/blob/trunk/packages/compose/src/hooks/use-preferred-color-scheme-style/index.native.js) to change the styles depending on the user's colour scheme. It's my understanding that, with this hook, styles created under the light mode selector will always be applied and then we can override the ones we need to using the dark mode selector. So, the border widths here should be inherited from the light mode selector by default and that seems to be working in testing. 

Let me know if that seems right to you or if you still think more is needed here!